src/main/java/org/jabref/logic/integrity/EditionChecker.java

```diff
@@ -66,4 +67,9 @@ public EditionChecker(BibDatabaseContext bibDatabaseContext, boolean allowIntege
 
         return Optional.empty();
     }
+    boolean isFirstCharDigit (String input){
+        char[] array = input.toCharArray();
+        return Character.isDigit(array[0]);
```

Comment:
There must be a different way. Can't the first letter of a string be accessed? Doesn't Google Guava offer a method for that?

Reply:
There is indeed a simpler way: string,charAt(0) 
https://beginnersbook.com/2013/12/java-string-charat-method-example/

Reply:
Yes in Guava the the most similar metod is CharMetcher.digit()  
[https://guava.dev/releases/snapshot/api/docs/com/google/common/base/CharMatcher.html#digit--](url)  
which is deprecated

Reply:
Simple use  Character.Digit(input.chartAt(0)) that is the easiest solution

Reply:
This is not done. I will do it manually after the merge.